src/commands/help_menu.py

```diff
@@ -0,0 +1,205 @@
+            )
+            for command in self.misc_commands:
+                embed.add_field(
+                    name=f"{self.capfirst(command.name)}",
+                    value=f"{command.description}",
+                    inline=False,
+                )
+
+        elif (
+            self.currentpage > 0 and self.currentpage < self.maxpages
+        ):  # Menu for different groups
+            command = self.group_commands[self.currentpage - 1]
+            # Adding fields for groups
+            embed = Embed(
+                title=f"{self.capfirst(command.name)} Commands",
+                description=f"{command.description}",
+                color=Color.yellow(),
+            )
+            for subcommand in command.commands:
+                if isinstance(subcommand, app_commands.Group):
+                    for subsubcommand in subcommand.commands:
+                        embed.add_field(
+                            name=f"/{command.name} {subcommand.name} {subsubcommand.name}",
+                            value=subcommand.description,
+                            inline=True,
+                        )
+                else:
+                    embed.add_field(
+                        name=f"/{command.name} {subcommand.name}",
+                        value=subcommand.description,
+                        inline=True,
+                    )
+        elif self.currentpage == self.maxpages:
+            embed = Embed(
+                title="Miscellaneous Commands",
+                description="These are commands that are not part of any group.",
+                color=Color.yellow(),
+            )
+            for command in self.misc_commands:
+                embed.add_field(
+                    name=f"/{command.name}",
+                    value=f"{command.description}",
+                    inline=False,
+                )
+        return embed
+
+    def total_pages(self):
+        """Calculate total pages: 1 for grouped commands, 1 for misc commands, and 1 for each group."""
+        return 2 + len(self.group_commands)
+
+    # Update the options in select menu
+    def update_select_options(self):
+        options = []
+        for i, command in enumerate(self.group_commands):
+            label = f"{command.name}"
+            options.append(
+                SelectOption(label=label, value=str(i), description=command.description)
+            )
+        options.append(
+            SelectOption(
+                label="Miscellaneous Commands",
+                value=self.maxpages,
+                description="These are commands that are not part of any group.",
+            )
+        )
+        for item in self.children:
+            if isinstance(item, ui.Select):
+                item.options = options
+
+    def capfirst(self, string):
+        newstring = string[0].upper() + string[1:].lower()
+        return newstring
+
+    """
+    buttons change currentpage to corresponding value
+    """
```

Comment:
Replace with a # comment
```suggestion
   # buttons change currentpage to corresponding value
```